### PR TITLE
Fix recording timer

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.9.1
- Fixed: Use correct start time for EPG lookup when adding timer
- Fixed: strncpy fix possible length fault

### CODE REVIEW

File: pvr.vuplus/addon.xml.in [hunk 1]
```diff
@@ -175,6 +175,10 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v4.9.1
+- Fixed: Use correct start time for EPG lookup when adding timer
+- Fixed: strncpy fix possiible length fault
```
Comment (ksooo) [inline pvr.vuplus/addon.xml.in]: Here is your humble (non-native English speaking) language reviewer. ;-)

typo: possiible (one `i` should be enough)

File: pvr.vuplus/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+v4.9.1
+- Fixed: Use correct start time for EPG lookup when adding timer
+- Fixed: strncpy fix possiible length fault
```
Comment (ksooo) [inline pvr.vuplus/changelog.txt]: typo: possiible

**Review by ksooo — COMMENTED** [review]
typos in changelog, everything else looks good

**Comment by phunkyfish** [comment]
Ok, ready now.

**Review by ksooo — APPROVED** [review]

**Review by ksooo — APPROVED** [review]

**Pull request merged**